extensions/ql-vscode/package.json

```diff
@@ -878,10 +878,6 @@
         }
       ],
       "commandPalette": [
-        {
```

Comment:
How come we're removing this command? Should we just remove the `when` clause?

Reply:
The default is for all commands to be visible to the user in the command palette. So, when a command is not in the command palette, it will always be visible to the user. For example, the `codeQL.chooseDatabaseLgtm` command is also not in the `commandPalette` section, only in the `commands` section.

Reply:
Ah awesome! Thanks for explaining - wasn't aware of that.